Source/WebCore/xml/XPathFunctions.cpp

```diff
@@ -706,7 +706,8 @@ double FunRound::round(double val)
 
 Value FunRound::evaluate() const
 {
-    return round(argument(0).evaluate().toNumber());
+    // Why does std::round not work here?
+    return FunRound::round(argument(0).evaluate().toNumber());
```

Comment:
No need to qualify the function in the same class so we should just keep the original code.

Reply:
I got this confused with std::round so yeah we should keep this.